tests/fixtures/testModel.ts

```diff
@@ -333,12 +422,6 @@ export class TestClassModel extends TestClassBaseModel {
   stringProperty: string;
   protected protectedStringProperty: string;
 
-  public static typeLiterals = {
```

Comment:
I removed these since they were messing up type resolution (`Boolean, Number and String`) will throw.